setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash 
+
+conda create -n xnat_downloader_env python=2.7
+source activate xnat_downloader_env
+conda install -c conda-forge dcm2niix
```

Comment:
This line can be removed if we create the environment like so: 
`conda create -n xnat_downloader_env2 python=2.7 dcm2niix -c conda-forge -y`